README.md

```diff
@@ -176,6 +176,7 @@ key is `urls`. Other optional options are:
   of strings for headless Chrome](https://peter.sh/experiments/chromium-command-line-switches/).
 * `cssoOptions` - CSSO compress function [options](https://github.com/css/csso#compressast-options)
 * `timeout` - Maximum navigation time in milliseconds, defaults to 30 seconds, pass 0 to disable timeout.
+* `ignoreCssError` - ignore CSS errors. By default minimalcss reports all CSS errors, but if you want to ignore those errors, if you do not have access to those css files, you can set this option to true.
```

Comment:
The ` - ignore CSS errors.` part isn't super useful. The functionality is pretty clear from the name of the option (`ignoreCssErrors`). 

I would change it to something like:

```
By default, any CSS parsing error throws an error in minimalcss. If you know it's safe to ignore (for example, third-party CSS resources), set this to true.
```